CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Using recent CMake versions is just convenient because of nice new features. You just spend less time programming CMake. Also testing older versions is difficult because CIs usually have very recent versions.